radixdlt/src/main/java/com/radixdlt/network/SimpleEventCoordinatorNetwork.java

```diff
@@ -0,0 +1,131 @@
+/*
+ *  (C) Copyright 2020 Radix DLT Ltd
+ *
+ *  Radix DLT Ltd licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License.  You may obtain a copy of the
+ *  License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  either express or implied.  See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.network;
+
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.subjects.PublishSubject;
+
+import java.util.Objects;
+import javax.inject.Inject;
+import org.radix.network.messaging.Message;
+import org.radix.network2.addressbook.AddressBook;
+import org.radix.network2.addressbook.Peer;
+import org.radix.network2.addressbook.PeerWithSystem;
+import org.radix.network2.messaging.MessageCentral;
+import org.radix.universe.system.LocalSystem;
+
+import com.radixdlt.common.EUID;
+import com.radixdlt.consensus.NewRound;
+import com.radixdlt.consensus.Vertex;
+import com.radixdlt.consensus.Vote;
+import com.radixdlt.consensus.messages.NewRoundMessage;
+import com.radixdlt.consensus.messages.VertexMessage;
+import com.radixdlt.consensus.messages.VoteMessage;
+import com.radixdlt.universe.Universe;
+
+/**
+ * Simple network that publishes messages to known nodes.
+ */
+public class SimpleEventCoordinatorNetwork implements EventCoordinatorNetworkSender, EventCoordinatorNetworkRx {
```

Comment:
Actual networking for Event Coordinator.

Reply:
Moved into networking package to separate from consensus proper.

Reply:
it is actually `EventCoordinatorSimpleNetwork`, right?

Reply:
Depends on your point of view I guess :)  For me it's an EventCoordinatorNetwork that's simple.